x-pack/plugins/ml/server/models/data_recognizer/modules/sample_data_flights/ml/mean_delay_by_dest_weather.json

```diff
@@ -0,0 +1,39 @@
+
+{
+  "groups": ["kibana_sample_data"],
+  "description": "Kibana sample flights data: detect anomalies in mean flight delay",
+  "analysis_config" : {
+    "bucket_span": "1h",
+    "detectors": [
+      {
+        "detector_description": "Mean flight delay time by airline carrier",
+        "function": "high_mean",
+	"field_name": "FlightDelayMin",
+	"partition_field_name": "Carrier"
+      }
+    ],
+    "influencers": ["Carrier", "DestWeather", "OriginWeather"]
+  },
+  "analysis_limits": {
+    "model_memory_limit": "10mb"
+  },
+  "data_description": {
+    "time_field": "timestamp"
+  },
+  "model_plot_config": {
+    "enabled": true
+  },
+  "custom_settings": {
+    "created_by": "ml-module-sample",
+    "custom_urls": [
+      {
+        "url_name": "Raw data",
+        "url_value": "kibana#/discover?_g=(time:(from:\u0027$earliest$\u0027,mode:absolute,to:\u0027$latest$\u0027))&_a=(index:d3d7af60-4c81-11e8-b3d7-01146121b73d)" 
```

Comment:
Did you consider passing `Carrier` to the Discover page? I wonder if it is more useful to show only the data for the selected partitioning field, rather than all carriers?

Reply:
Fixed!